--- a/FE/src/components/Service/Service.jsx
+++ b/FE/src/components/Service/Service.jsx
@@ -16,7 +16,7 @@ const Service = () => {
   const [selectedServices, setSelectedServices] = useState([]);
   const [isLoading, setIsLoading] = useState(false);
   const [isCardView, setIsCardView] = useState(true);
-  const { control, handleSubmit, reset, formState: { errors } } = useForm();
+  const { control, handleSubmit, reset, formState: { errors } } = useForm({});
   const navigate = useNavigate();
   const [page, setPage] = useState(1);
   const [pageSize, setPageSize] = useState(10);
@@ -31,7 +31,7 @@ const Service = () => {
   const { data: userInfo, isLoading: isFetchingUser } = useQuery({
     queryKey: ['userInfo', currentUser?._id],
     queryFn: async () => {
-      const { data } = await instance.get(`/user/${currentUser?._id}`);
+      const { data } = await instance.get(`/user/${currentUser._id}`);
       return data.data;
     },
     enabled: !!currentUser,
